__tests__/app.test.js

```diff
@@ -328,3 +326,34 @@ describe("/api/articles/:article_id/comments", () => {
     });
   });
 });
+
+describe("/api/comments/:comment_id", () => {
+  describe("DELETE", () => {
+    test("DELETE 204: Deletes the specified comment by comment_id and returns status 204 with no content", () => {
+      return request(app)
+      .delete("/api/comments/5")
+      .expect(204)
+      .then(({ body }) => {
+        expect(body).toEqual({})
+      })
```

Comment:
I don't think you _need_ to worry about the body, as expecting 204 is enough to say that something that was there has been removed 😄